files-community/Files
# Code Quality: Introduced PathBreadcrumbViewModel

## Summary

I could not find the reason why PathBreadcrumb had never exist and I found it'd better add it. However, looking into the way of referencing to AddressToolbarViewModel and PathBreadcrumbView is too far from MVVM concept. I think we need to change it in the future.

## Task Checklist

- [x] Introduced PathBreadcrumbViewModel
- [x] De-clutter AddressToolbarViewModel

## Steps To Validate Changes

1. Launch the app
2. Go to Desktop
3. See the path breadcrumb items are changed accordingly and proeprly
4. Add a new tab
5. Change tabs
6. See the path breadcrumb items are changed accordingly and properly

## Known Issues

_N/A_

## PR Checklist

- [ ] **Development**: _N/A_

## Screenshots

_N/A_

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.
